src/providers/postgres/qgspostgresprovider.cpp

```diff
@@ -403,6 +403,17 @@ static bool operator<( const QVariant &a, const QVariant &b )
 }
 #endif
 
+#if QT_VERSION >= 0x050000 && QT_VERSION < 0x050600
```

Comment:
I was referring to this: https://github.com/qgis/QGIS/blob/master/src/providers/postgres/qgspostgresprovider.cpp#L322